test/tests.zig

```diff
@@ -473,6 +483,15 @@ pub fn addPkgTests(
         if (skip_non_native and !test_target.target.isNative())
             continue;
 
+        if (!is_wasmtime_enabled) {
```

Comment:
I believe you do not actually need this check. The build system is smart enough to know whether it should execute the tests or merely cross compile them based on the `enable_wasmtime` flag which you are setting below. By deleting this check, you make it so that the tests will at least test cross compiling the wasi tests, even if they cannot be executed.

Reply:
Good suggestion, thanks! Originally, I had it there because some behaviour tests were failing when targeting `wasm32`, but I've now narrowed it down to a few so this check is now indeed obsolete. See 2375df2 for changes.